index.js

```diff
@@ -9,14 +9,13 @@ module.exports = {
   included: function(app) {
     this._super.included(app);
     if (process.env.EMBER_CLI_FASTBOOT !== 'true') {
+      // In nested addons, app.bowerDirectory might not be available
+      var bowerDirectory = app.bowerDirectory || 'bower_components';
+      // In ember-cli < 2.7, this.import is not available, so fall back to use app.import
+      var importShim = typeof this.import !== 'undefined' ? this : app;
```

Comment:
A more compatible solution is to backport the import method used in ember-cli 2.7, as we've done in the up coming V4 release: https://github.com/ivanvanderbyl/ember-d3/blob/2b97ee97b60a33e9f479a093cc7ffb5e9aecbc7f/index.js#L33-L40